cl/recap/tasks.py

```diff
@@ -508,21 +483,16 @@ def process_recap_docket(self, pk):
         text = pq.filepath_local.read().decode()
     except IOError as exc:
         msg = f"Internal processing error ({exc.errno}: {exc.strerror})."
-        if (self.request.retries == self.max_retries) or pq.debug:
-            mark_pq_status(pq, msg, PROCESSING_STATUS.FAILED)
-            return None
-        else:
-            mark_pq_status(pq, msg, PROCESSING_STATUS.QUEUED_FOR_RETRY)
-            raise self.retry(exc=exc)
+        await mark_pq_status(pq, msg, PROCESSING_STATUS.FAILED)
+        return None
 
     if "History/Documents" in text:
         # Prior to 1.1.8, we did not separate docket history reports into their
         # own upload_type. Alas, we still have some old clients around, so we
         # need to handle those clients here.
         pq.upload_type = UPLOAD_TYPE.DOCKET_HISTORY_REPORT
-        pq.save()
-        process_recap_docket_history_report(pk)
-        self.request.chain = None
+        await pq.asave()
+        await process_recap_docket_history_report(pk)
```

Comment:
We can yank this while we're here. Those clients are OLD, and I just don't imagine they still even exist.

Reply:
removed this codepath